test: rename multithreaded tests prefix to "mtt-"

grom72 opened the pull request:
Use shorter multithreaded tests prefix "mtt-", unified with unit tests prefix "ut-"

e.g.
$ ctest
...
 74/169 Test  #74: ut-utils-ibv_context_is_odp_capable_0_none .............................   Passed    0.03 sec
        Start  75: ut-log_default-init-fini_0_none
 75/169 Test  #75: ut-log_default-init-fini_0_none ........................................   Passed    0.03 sec
        Start  76: ut-log_default-function_0_none
 76/169 Test  #76: ut-log_default-function_0_none .........................................   Passed    0.03 sec
        Start  77: mtt-conn_cfg-get_cq_size_0_none
 77/169 Test  #77: mtt-conn_cfg-get_cq_size_0_none ..............................   Passed    0.03 sec
...

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/pmem/rpma/1908)
<!-- Reviewable:end -->

haichangsi reviewed (commented):
> Reviewed 1 of 1 files at r1, 1 of 1 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/pmem/rpma/1908)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @grom72)



<!-- Sent from Reviewable.io -->

haichangsi reviewed (commented):
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>

> *[Reviewable](https://reviewable.io/reviews/pmem/rpma/1908#-:-N5t_81y1LJX2d7zUtpW:bnfp4nl)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @grom72)



<!-- Sent from Reviewable.io -->

osalyk reviewed (changes requested):
> Reviewed 1 of 1 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/pmem/rpma/1908)* status: all files reviewed, 1 unresolved discussion (waiting on @grom72)

*[a discussion](https://reviewable.io/reviews/pmem/rpma/1908#-N5tqTaX12WMnlpJZvvO:-N5tqTaX12WMnlpJZvvP:bl5kkic) (no related file):*
Please correct the number of commits

___


<!-- Sent from Reviewable.io -->

grom72 committed:
test: rename multithreaded tests prefix to "mtt-"

Use shorter multithreaded tests prefix "mtt-",
unified with unit tests prefix "ut-"

grom72 reviewed (commented):
> *[Reviewable](https://reviewable.io/reviews/pmem/rpma/1908)* status: 1 of 2 files reviewed, 1 unresolved discussion (waiting on @haichangsi and @osalyk)

*[a discussion](https://reviewable.io/reviews/pmem/rpma/1908#-N5tqTaX12WMnlpJZvvO:-N67l4CF-s1YrMBRNAHA:b-896fix) (no related file):*
<details><summary><i>Previously, osalyk (Oksana Sałyk) wrote…</i></summary><blockquote>

Please correct the number of commits
</blockquote></details>

Done.

___


<!-- Sent from Reviewable.io -->

osalyk reviewed (approved):
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>

> Reviewed all commit messages.
> *[Reviewable](https://reviewable.io/reviews/pmem/rpma/1908#-:-N67n4Ws2Fhtk3pr9E2l:bnfp4nl)* status: 1 of 2 files reviewed, all discussions resolved (waiting on @haichangsi)



<!-- Sent from Reviewable.io -->

Patryk717 reviewed (commented):
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/> 

> *[Reviewable](https://reviewable.io/reviews/pmem/rpma/1908#-:-N6HwZ8D7ss4tfT3ESLX:bg7igkv)* status: 1 of 5 files reviewed, all discussions resolved (waiting on @haichangsi and @osalyk)



<!-- Sent from Reviewable.io -->

osalyk reviewed (approved):
> Reviewed 1 of 1 files at r3, 3 of 3 files at r5, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/pmem/rpma/1908)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @grom72)



<!-- Sent from Reviewable.io -->
